packages/web-twig/tests/components-fixtures/breadcrumbsDefault.twig

```diff
@@ -0,0 +1,20 @@
+{% set items = [
+    {
+        title: 'Root',
+        url: '#rootUrl',
+    },
+    {
+        title: 'Category',
+        url: '#categoryUrl',
+    },
+    {
+        title: 'Subcategory',
+        url: '#subcategoryUrl',
+    },
+    {
+        title: 'Current page',
+        url: '#currentUrl',
+    },
+] %}
+
+<Breadcrumbs />
```

Comment:
You are not passing an `items` prop. And it is working just because you are using `items` instead of `_items` variable in a Breadcrumbs implementation.

Reply:
Fixed